example/grpc-web/README.md

```diff
@@ -0,0 +1,62 @@
+# Description
+The grpc-web example shows how to use the Dart gRPC library with a gRPC-Web capable server. 
+
+This is meant to be used with the echo example provided by the grpc-web repository. The definition of the service is given in echo.proto. 
+
+# Prerequistes
+You will need a clone of the grpc-web repository to run the example server.
+
+You will also need the dart 'webdev' tool, which you can get by running:
+
+```sh
+$ pub global activate webdev
+```
+
```

Comment:
Good call -- you need to run the server yes, but I'm thinking I should just delegate those instructions to whatever is in the grpc-web base repository, in case they change.  I will make that portion of the instrucitons more clear though.